### PR TITLE
Pass `atol` parameter to FITSDiff

### PR DESCRIPTION
**Pull request by svank**
I noticed that the `atol` parameter doesn't get passed to `astropy.io.fits.diff.FITSDiff` when the reference file is FITS, and so setting `atol` has no effect. (e.g. `@pytest.mark.array_compare(file_format='fits', atol=1)` still used an `atol` of 0.) This PR adds a test for `atol`, ensures `atol` is passed through, and mentions the `atol` parameter in the readme alongside `rtol` (which was already mentioned).

As I was adding the test, I noticed that the existing test for the `rtol` parameter was buggy, in that the generated and reference files both contained only values of 1.6, so `rtol` could be zero or any other value and the test would still pass. (Maybe the reference files were regenerated at some point, forgetting that in this particular case, the reference file must be different from the generated file to test the tolerance values.) I've modified that test by scaling up the generated values so that the test fails if `rtol` isn't set to a large value, to ensure that the value of `rtol` is being used.

### CODE REVIEW

**Review by pllim — COMMENTED** [review]
I don't grok the `reference_dir` setup. Is there no way to generate the truth in memory at run time? Seems weird to have to copy the same file over for a new test.

**Comment by svank** [comment]
Not sure if this answers your question, but since these tests are testing the machinery that compares generated data to a reference file, I think there has to be a reference file to compare to, even if only trivial data is being generated in this case. And since the reference file name is autogenerated from the test name, I think a new test needs a new copy of the file. I suppose the reference files could be programmatically generated in a temp dir as the tests run, but I'm not sure how to easily pass that temp dir to the `pytest-arraydiff` wrappers.

I could easily believe I'm missing a better way to do this (or misunderstanding the tests), in this weird context where the testing framework is being used to test the testing framework itself.

**Comment by pllim** [comment]
You are probably doing it right. I am not familiar with this package. 🤷

**Comment by pllim** [comment]
I had to re-enable CI in the settings. It was disabled due to inactivity. So I have to close/re-open to kick it off. Please do not be alarmed.

**Comment by svank** [comment]
Done!

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 
 - Add ability to compare to Pandas DataFrames and store them as HDF5 files [#23]
 
+- Fix ``array_compare`` so that the ``atol`` parameter is correctly used with
+  FITS files.
```
Comment (pllim) [inline CHANGES.md:7]:
```suggestion
  FITS files. [#33]
```

Comment (svank) [reply CHANGES.md]: Added, thanks!

**Review by astrofrog — APPROVED** [review]

**Comment by pllim** [comment]
Thanks, all!

**Pull request merged**